database/bulkSeedMon.js

```diff
@@ -0,0 +1,51 @@
+const faker = require('faker');
+const generator = require('../helpers/generator.js');
+const db = require('./index.js');
+
+const generateData = () => {
+  return new Promise((resolve) => {
+    const descriptions = [];
+    for (let i = 0; i < 100000; i++) {
+      const noun = generator.genNoun();
+      const numBedrooms = generator.genNumBedrooms(noun);
+      const numGuests = generator.genNumGuests();
+      descriptions.push({
+        host: {
+          _id: i,
+          name: faker.name.firstName(),
+          pic: faker.image.avatar()
+        },
+        title: generator.genTitle(noun),
+        city: generator.genCity(),
+        numGuests,
+        homeType: generator.genHomeType(noun),
+        numBedrooms,
+        numBeds: generator.genNumBeds(numGuests),
+        numBaths: generator.genNumBaths(),
+        description: generator.genDescription(noun, numBedrooms)
+      });
+    }
+    console.log('one: return descriptions');
+    resolve(descriptions);
+  });
+};
+
+const insert = (houses) => {
+  return new Promise((resolve) => {
```

Comment:
FYI, if you are using **insertMany** from mongoose, **insertMany** return a promise. So that you can just use await keyword in **insertMany**.
https://mongoosejs.com/docs/api.html#model_Model.insertMany